### DISCUSSION TITLE
Collapsed Sections use resources?

### DISCUSSION THREAD

**Discussion by brpeterso** [Lissy93/dashy#692] [Help and Support] [answered]
Just curious, if I set my widgets to collapsed, will they not pull data until I open them?  ...or do they pull resources all the time, even when collapsed?  I'm just trying to see if it speeds up load times and potentially reduces the times that I see the "Oh Snap" error message in Chrome when I come back to my dashy homepage.

Thanks!

**Comment by lissy93** [comment, marked as answer]
Yeah, currently data is still fetched, even when sections are collapsed.
I'll look into improving that flow in the future, Re: https://github.com/Lissy93/dashy/issues/643#issuecomment-1122496097
